File: tests/machine/x/fs/slice.fs

printfn "%s" (String.concat " " (List.map string [1; 2; 3].[1..(3-1)]))
printfn "%s" (String.concat " " (List.map string [1; 2; 3].[0..(2-1)]))
printfn "%s" "hello".Substring(1, 4 - 1)

File: tests/transpiler/x/pl/slice.pl
:- style_check(-singleton).
:- initialization(main).

main :-
    writeln(2),
    writeln(3),
    writeln(1),
    writeln(2),
    L2 is 4 - 1, sub_string("hello", 1, L2, _, R2), writeln(R2).
